### DISCUSSION TITLE
Preguntas recuperativas

### DISCUSSION THREAD

**Discussion by cbuerger02** [IIC2133-PUC/2024-1#195] [Ayuda] [answered]
Hola, 

una consulta, para las preguntas recuperativas después de la I3, existe la posibilidad de llevar un torpedo con la materia respectiva? También, como será el formato de esa prueba? Es como si dieramos la prueba que faltamos de nuevo, es más corta, etc.? Gracias de antemano!

Saludos,
Christoph

**Comment by Dafnemami** [comment, marked as answer]
Hola! En la clase de repaso I3 que tuvieron se indicó lo siguiente:![image](https://github.com/IIC2133-PUC/2024-1/assets/53848830/c2b7239c-3069-4200-bd96-65cfc9b2eda1)

Un alcancé sobre lo anterior: Es 1h20min y no 1h como se indica en la slide.

Espero aclare tu duda

**Reply by cbuerger02** [reply]
Muchas gracias! Todo super claro
